presentation/frontend/package.json

```diff
@@ -23,10 +24,14 @@
   },
   "scripts": {
     "compile": "npm run -s build",
-    "copy:locale": "cpx \"../common/assets/**/*.json\" ./lib/public",
-    "copy:assets": "cpx \"./src/**/*.json\" ./lib",
-    "build": "npm run -s copy:assets && npm run -s copy:locale && npm run -s pseudolocalize && npm run -s extract && tsc 1>&2",
-    "build:watch": "tsc -w",
+    "build": "npm run -s build:cjs && npm run -s build:esm",
+    "build:cjs": "npm run -s copy:cjs && tsc -p tsconfig.cjs.json",
+    "build:esm": "npm run -s copy:esm && tsc -p tsconfig.esm.json",
+    "copy:cjs": "cpx \"../common/assets/**/*.json\" ./lib/cjs/public && cpx \"./src/**/*.json\" ./lib/cjs",
```

Comment:
Similar to what we had in `components` - for some reason the script is copying assets from another package. That should not be necessary.